frontend/packages/console-dynamic-plugin-sdk/src/extensions/actions.ts

```diff
@@ -95,8 +97,10 @@ export const isActionFilter = (e: Extension): e is ActionFilter => {
 export type Action = {
   /** A unique identifier for this action. */
   id: string;
-  /** The label to display in the UI. */
+  /** The translated label to display in the UI. */
   label: string;
+  /**  To send translation key for label */
+  labelKey?: string;
```

Comment:
I don't think we need this prop. Since all new extensions are hook based, strings will be translated in the hook itself